speech/setup.py

```diff
@@ -51,6 +51,8 @@
 
 REQUIREMENTS = [
     'google-cloud-core >= 0.20.0',
+    'gapic-google-cloud-speech-v1beta1==0.11.1',
+    'grpc-google-cloud-speech-v1beta1==0.11.1',
```

Comment:
Instead of hardcoding this, use a looser bound: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/0f8fdb6d2360e937dfba395b142de42c618eb228/pubsub/setup.py#L56-L57

@bjwatson is the `0.11.x` series the right thing to target?